src/fprime/fbuild/cmake.py

```diff
@@ -107,7 +107,7 @@ def execute_known_target(
             module = self.get_cmake_module(path, build_dir)
             cmake_target = (
                 module
-                if target == ""
```

Comment:
This one is another where I am hesitant.  The purpose of this call was to check for exactly empty string.  In this system targets are of the form:

1. target, for top level targets
2. module_target, for module targets
3. module, special case, only when target is exactly ""